packages/cloud-cognitive/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -39,9 +39,10 @@ export let UserProfileImage = React.forwardRef(
     {
       backgroundColor,
       className,
-      icon,
+      kind,
       initials,
       image,
+      imageDescription,
```

Comment:
You'll need to add this to your props validation below, with some documentation/comments on how this prop is used! Additionally, because it provides an `alt` for the image, it should likely be required.